MailerQ/Core/QueueManager.cs

```diff
@@ -30,6 +30,29 @@ public QueueManager(MailerQConfiguration configuration)
 
         public QueueManager(IOptions<MailerQConfiguration> options) : this(options.Value) { }
 
+        public void DeclareDeadLetterQueueForPublish(
+            string name,
+            bool addQueueNamePrefix = true,
+            bool durable = true,
+            int messageTtl = 10,
+            int maxPriority = (int)Priority.High,
+            string deadLetterExchange = "",
+            string deadLetterRoutingKey = QueueName.Outbox)
+        {
+            if (addQueueNamePrefix)
+            {
+                name = $"{configuration.QueuesNamePrefix}{name}";
+            }
+
+            bus.QueueDeclare(
+                name: name,
+                durable: durable,
+                perQueueMessageTtl: messageTtl,
+                maxPriority: maxPriority,
+                deadLetterExchange: deadLetterExchange,
+                deadLetterRoutingKey: deadLetterRoutingKey);
```

Comment:
What happens if this is called and the queue already exists? and if the queue exists with different parameters?

Reply:
This is the Summary of `QueueDeclare`, if exist do nothing

```
        //
        // Summary:
        //     Declare a queue. If the queue already exists this method does nothing
        //
```